mmocr/models/textrecog/losses/ctc_loss.py

```diff
@@ -54,6 +59,16 @@ def forward(self, outputs, targets_dict):
                 targets[idx, :valid_len] = tensor[:valid_len]
 
         target_lengths = targets_dict['target_lengths']
+        target_lengths = torch.clamp(target_lengths, min=1, max=seq_len).long()
+
+        input_lengths = torch.full(
+            size=(bsz, ), fill_value=seq_len, dtype=torch.long)
+        if not self.flatten and valid_ratios is not None:
```

Comment:
why self.flatten=False?  How about if it is true?

Reply:
If flatten is True, `input_lengths` must be filled with the value `seq_len` to use cudnn.